### ISSUE TITLE
type: "Exam" triggers "wait for proctor" regardless of access controls set with mode: "public".

### ISSUE THREAD

**Issue by coatless** [PrairieLearn/PrairieLearn#1649]
Example configuration:

```json
{
    "uuid": "b02923c2-c454-11e9-b6a6-0ae03cdc714c",
    "type": "Exam",
    "title": "Practice Homework",
    "set": "Homework",
    "number": "1",
    "allowIssueReporting": true,
    "allowAccess": [
        {
            "mode": "Public",
            "role": "TA",
            "credit": 100
        },
        {
            "mode": "Public",
            "credit": 100,
            "startDate": "1900-07-07T23:59:59",
            "endDate": "2300-07-10T23:59:59"
        }
    ],
    "zones": [
        {
            "title": "Template Questions",
            "questions": [
                {"id": "template-checkbox", "points": [10, 10, 10, 5, 5]},
                {"id": "template-mc", "points": 10},
                {"id": "template-numeric", "points": 10}
            ]
        }
    ],
    "text": "This is some text."
}
```

Generates:

<img width="822" alt="Screen Shot 2019-08-21 at 7 24 55 PM" src="https://user-images.githubusercontent.com/833642/63477132-61901500-c449-11e9-95cf-a20235accc83.png">

Relevant hard coding:

https://github.com/PrairieLearn/PrairieLearn/blob/77e142b1d71254e6f68eacb46500b20df289b001/pages/studentAssessmentExam/studentAssessmentExam.ejs#L17

**Comment by trombonekenny** [comment]
IMO, this doesn't need a configuration option to toggle. - we just need to check for "mode": "Exam" before showing the "Please wait" text.

**Comment by mwest1066** [comment]
@trombonekenny I see where you are going with this, but for practice exams I’d kind of like to have them be as identical as possible to the real exams. I’ve never had students be confused by this message on a practice exam, perhaps because it’s clear that this is practice for the real thing.

I can see that if there is never going to be an exam in the CBTF, so the Public exam is the only copy, then the “please wait” message doesn’t make sense and so an option to remove it would be reasonable.

**Comment by trombonekenny** [comment]
I'm just trying to avoid config fatigue. If you think there are cases that need us to be able to toggle between them, then I'm on-board.

**Comment by trombonekenny** [comment]
Closed by #1654. Thanks @coatless!
